### DISCUSSION TITLE
Camera: How to skip photo validation step ?

### DISCUSSION THREAD

**Discussion by Aarbel** [ionic-team/capacitor#5008] [Q&A]
After taking a picture, user need to validate it.

How to skip this step ? 

![Uploading image.png…]()

**Comment by jcroucher** [comment]
I am doing that with this plugin

https://github.com/capacitor-community/camera-preview

**Reply by Aarbel** [reply]
@jcroucher thanks a lot, do you have a code preview to explain more how you handle Camera WebRTC camera without having a validation step
